elektronLogging/createLogDir.go

```diff
@@ -0,0 +1,38 @@
+package elektronLogging
+
+import (
+	"os"
+	"strconv"
+	"time"
+	logrus "github.com/sirupsen/logrus"
+)
+
+var logDir string
+
+func GetLogDir(startTime time.Time, prefix string) {
+	if logDir == "" {
+		logDir = createLogDir(prefix, startTime)
+	}
+}
+
+func createLogDir(prefix string, startTime time.Time) string {
+	// Creating directory to store all logs for this run
+	logDirName := "./" + prefix + strconv.Itoa(startTime.Year())
+	logDirName += "-"
```

Comment:
Why not use `string.join([]string{...}, "")` ?

Reply:
Yeah, thats right